R/dist_negative_binomial.R

```diff
@@ -3,8 +3,9 @@
 #' \lifecycle{stable}
 #'
 #' A generalization of the geometric distribution. It is the number
-#' of successes in a sequence of i.i.d. Bernoulli trials before
-#' a specified number (\eqn{r}) of failures occurs.
+#' of failures in a sequence of i.i.d. Bernoulli trials before
+#' a specified number of success (`size`) occurs. The probability of success in
```

Comment:
```suggestion
#' a specified number of successes (`size`) occurs. The probability of success in
```

Reply:
I'm not sure about the verb actually... if it goes with "number" or "successes".  But the original had "failures" so I'd go with plural.